__tests__/games.test.js

```diff
@@ -170,4 +170,64 @@ describe("PATCH: /api/reviews/:review_id", () => {
             expect(body.msg).toBe("bad request")
         })
     })
+})
+
+describe("GET: /api/reviews/:review_id", () => {
+    it("200: should respond with an object of review of correct id with a comment_count", () => {
+        return request(app)
+        .get("/api/reviews/3")
+        .expect(200)
+        .then(({body}) => {
+            expect(typeof body.review).toBe("object")
+            expect(body.review).toMatchObject({
+                    title: 'Ultimate Werewolf',
+                    designer: 'Akihisa Okui',
+                    owner: 'bainesface',
+                    review_img_url:
+                      'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                    review_body: "We couldn't find the werewolf!",
+                    category: 'social deduction',
+                    created_at: expect.any(String),
+                    votes: 5,
+                    comment_count: 3
+            })
+        })
+    })
+    it("200: should respond with an object of the correct review with the comment count at 0", () => {
+        return request(app)
+        .get("/api/reviews/8")
+        .expect(200)
+        .then(({body}) => {
+            expect(typeof body.review).toBe("object")
+            expect(body.review).toMatchObject({
+                title: 'One Night Ultimate Werewolf',
+                designer: 'Akihisa Okui',
+                owner: 'mallionaire',
+                review_img_url:
+                  'https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260',
+                review_body: "We couldn't find the werewolf!",
+                category: 'social deduction',
+                created_at: expect.any(String),
+                votes: 5,
+                comment_count: 0
+            })
+        })
+    })
+    it("404: should respond with not found when an invalid id is passed", () => {
+        return request(app)
+        .get("/api/reviews/100000")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("not found")
+        })
+    })
+    it("400: should respond with bad request when an invalid request is passed", () => {
```

Comment:
This test seems like it's actually testing for anon-existent path (/api/revi3ws/3) rather than a bad request. A bad request would come from the user e.g. they pass something like notANumber for the article id. Please refactor this.

Reply:
My error handling is working but I wrote the test wrong and it was failing when I tried it again. Not sure how I missed it but It should be working now